src/table/utils/__tests__/handle-scroll.spec.js

```diff
@@ -53,22 +53,33 @@ describe('handle-scroll', () => {
       expect(evt.preventDefault).not.have.been.called;
     });
 
-    it('should not run preventDefault when not scroll to rightmost place in ltr direction', () => {
+    it('should not run preventDefault when scroll to the leftmost place but not keep scrolling left in LTR direction', () => {
+      isRTL = false;
       evt = {
         ...evt,
-        deltaX: 10,
+        deltaX: 0,
+      };
+      handleHorizontalScroll(evt, isRTL, memoedContainer);
+      expect(evt.stopPropagation).have.been.calledOnce;
+      expect(evt.preventDefault).not.have.been.called;
+    });
+
+    it('should not run preventDefault when scroll to the rightmost place but not keep scrolling right in LTR direction', () => {
+      evt = {
+        ...evt,
+        deltaX: 0,
       };
       memoedContainer = {
         ...memoedContainer,
-        scrollLeft: 400,
+        scrollLeft: 1369,
```

Comment:
No point of having uneven numbers here, it just makes the tests harder to understand.